docs/docs/overview/quickstart/get-started-with-node.js.md

```diff
@@ -0,0 +1,244 @@
+---
+sidebar_position: 5
+sidebar_label: Get started with Node.js
+---
+
+# Node.js Quickstart
+
+Welcome to the Node.js Quickstart guide for Novu, a powerful notification service that enables you to send multi-channel (SMS, Email, Chat, Push) notifications from your Node.js applications. In this Quickstart, you'll learn how to seamlessly integrate Novu into your app and perform various essential tasks. Let's get started!
+
+## **Prerequisites**
+
+Before diving into the Quickstart, make sure you have the following:
+
+- Node.js installed on your development machine.
+- A Novu account. If you don't have one, sign up for free at "https://web.novu.co"
+
+### **Install and Set Up Novu in your Node.js App**
+
+First, you must install the Novu package in your Node.js application. Open your terminal and run the following command:
+
+```bash
+npm install @novu/node
+```
+
+Once installed, you can import Novu into your app and initialize it using your Novu account credentials. This step establishes a connection between your app and the Novu notification service.
+
+```jsx
+import { Novu } from '@novu/node';
+
+const novu = new Novu('<YOUR_NOVU_API_KEY>');
+```
+
+Replace the `<YOUR_NOVU_API_KEY>` value with the authentic key from the **API Key** section of your [Novu Dashboard](https://web.novu.co/settings).
+
+<aside>
+🔑 **Note:** Please do not hardcode your credentials in a file in production. Use environment variables instead.
+
+</aside>
+
+## Set Up A Channel Provider
+
+A channel provider is a service that provides one or more notification functionality such as sending an email, SMS, push notification etc. Our [integration store](https://web.novu.co/integrations) includes four channels: Email, SMS, Chat, and Push. These channels have multiple providers associated with them.
+
+| Channel | Providers                                                           |
+| ------- | ------------------------------------------------------------------- |
+| Email   | MailGun, Mandrill, MailJet, Amazon SES, Sendgrid, Postmark, Netcore |
+| SMS     | Twilio, Amazon SNS, Plivo, SMS, SMSCentral, Kannel, Infobip, Termii |
+| Chat    | Mattermost, Slack, Microsoft Teams, Discord                         |
+| Push    | FCM, APNS, Expo                                                     |
+
+Only one provider can be **active** per **channel**. Connect any of your favorite providers to get started. The email channel comes with Novu's email provider, which is active by default and includes 300 credits.
+
+## Create A Notification Template
+
+A notification template is the blueprint for the notifications that will be sent. It holds the entire flow of messages sent to the subscriber. This is where all the different channels are tied together under a single entity.
+
+The template includes the following:
+
+- Notification template name and Identifier
```

Comment:
Notification workflow name and Identifier

Reply:
Done